Overview.html

```diff
@@ -866,6 +866,11 @@ <h4 id="requests"><span class="secno">3.1.5 </span>Requests</h4>
 <p>A <a href="#concept-request" title="concept-request">request</a> has an associated
 <dfn id="synchronous-flag">synchronous flag</dfn>. Unless stated otherwise it is unset.
 
+<p>A <a href="#concept-request" title="concept-request">request</a> should have an associated
+optional <dfn id="recursive-flag">recursive flag</dfn> set when
+<a href="#concept-main-fetch">main fetch</a> has been invoked recursively from
+<a href="#http-redirect-fetch">redirect fetch</a>. Unless stated otherwise it is unset.
```

Comment:
Yeah, this is unnecessary, it only needs to be listed at the definition of "main fetch" (perhaps with a "Note: the recursive flag is set when main fetch has been invoked recursively" (don't mention "from redirect fetch" anywhere, that may not always be the case)